src/server.tsx

```diff
@@ -241,6 +286,19 @@ app.get(
     },
 );
 
+app.get(
+    '/banner.js',
+    async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        try {
+            const module = await fs.promises.readFile(__dirname + '/../dist/modules/Banner.js');
```

Comment:
This has been updated/fixed - see https://github.com/guardian/contributions-service/blob/master/src/server.tsx#L235.